src/observability/summarizer.go

```diff
@@ -40,14 +40,13 @@ func extractNetworkInfoFromSystemLog(netLog pb.Alert, pods []types.Pod, services
 		}
 	} else if strings.Contains(netLog.Resource, "AF_UNIX") {
 		var path string
-		protocol = "AF_UNIX"
 		resslice := strings.Split(netLog.Resource, " ")
 		for _, locres := range resslice {
 			if strings.Contains(locres, "sun_path") {
 				path = strings.Split(locres, "=")[1]
 				if path != "" {
 					ip = path
-					bindAddress = path
+					protocol = "AF_UNIX"
```

Comment:
@yasin-cs-ko-ak This does not seem like a fix. We need to fix/remove the code carefully so that AF_NETLINK/AF_UNIX shows for proper values.

Reply:
@seswarrajan that value only shows for ingress/egress. In bind it is only AF_INET and AF_INET6. 
or we can change the AF_UNIX to UNIX as it is before.

Reply:
that AF_UNIX is not interfere with bind anymore.

Reply:
I understand that AF_UNIX and AF_NETLINK should not be present in the code itself and needs to be removed. 
We can discuss once.